src/main/java/net/minecraftforge/fml/client/gui/screen/ModListScreen.java

```diff
@@ -450,7 +450,7 @@ public void updateDynamicTexture() {
 
         lines.add(selectedMod.getDisplayName());
         lines.add(ForgeI18n.parseMessage("fml.menu.mods.info.version", MavenVersionStringHelper.artifactVersionToString(selectedMod.getVersion())));
-        lines.add(ForgeI18n.parseMessage("fml.menu.mods.info.idstate", selectedMod.getModId(), ModList.get().getModContainerById(selectedMod.getModId()).
+        lines.add(ForgeI18n.parseMessage("fml.menu.mods.info.idstate", selectedMod.getModId(), net.minecraftforge.fml.ModList.get().getModContainerById(selectedMod.getModId()).
```

Comment:
Couldn’t this be an import instead of a qualified reference?

Reply:
Because I renamed the ModListWidget to just ModList, which is now I duplicate name. I will revert it to ModListWidget to avoid confusion

Reply:
And this is a indication that you shouldn't of named it that.